cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/CleanCommandTest.java

```diff
@@ -87,39 +94,41 @@ public void testCleanCommandInProjectWithCustomTarget() throws IOException {
         Assert.assertTrue(Objects.requireNonNull(
                 customTargetDir.resolve("report").toFile().listFiles()).length > 0);
 
-        CleanCommand cleanCommand = new CleanCommand(projectPath, false, customTargetDir);
+        CleanCommand cleanCommand = new CleanCommand(projectPath, printStream, false, customTargetDir);
         cleanCommand.execute();
 
         Assert.assertFalse(Files.exists(customTargetDir));
     }
 
     @Test(description = "Test clean command on a ballerina project with custom target dir.")
-    public void testCleanCommandNonExistentTarget() {
-        Path projectPath = this.testResources.resolve("validProjectWithTarget");
+    public void testCleanCommandNonExistentTargetAndGenerated() throws IOException {
```

Comment:
Let's add a separate test case to verify the clean command when both target and generated directories are present.

Reply:
This is already tested in the test `testCleanCommandInProject` line 60